### DISCUSSION TITLE
Index reorganisation

### DISCUSSION THREAD

**Discussion by Petemac21** [olahallengren/sql-server-maintenance-solution#439] [Questions & Answers]
In the SQL Server Index and Statistics Maintenance solution here:

https://ola.hallengren.com/sql-server-index-and-statistics-maintenance.html

it states that "an index reorganization is not always possible".  Under what circumstances would an index reorganisation not be possible?

**Comment by olahallengren** [comment]
ALTER INDEX ... REORGANIZE is not supported in the following scenarios:

1. Page locks is disabled on the index, sys.indexes.allow_page_locks = 0

2. It is a memory optimized table, sys.tables.is_memory_optimized = 1
